src/cfn/stackArgsToX.ts

```diff
@@ -0,0 +1,90 @@
+import * as aws from 'aws-sdk';
```

Comment:
`s/stackArgsToX/inputHandling/` or something?

Reply:
Any other `something` suggestions? I couldn't think of any so the current name was my punt.